generators/entity-client/files.js

```diff
@@ -462,6 +480,12 @@ function writeFiles() {
 
             const entityTemplatesDir = `entity-client/templates/${templatesDir}`;
             this.writeFilesToDisk(files, this, false, this.fetchFromInstalledJHipster(entityTemplatesDir));
+
+            if (this.cypressTests) {
+                const cypressTemplatesDir = 'entity-client/templates';
```

Comment:
Move cypress files from `entity-client/templates/` to `entity-client/templates/common/`.
```suggestion
                const cypressTemplatesDir = 'entity-client/templates/common';
```